sari/.htaccess

```diff
@@ -9,15 +9,15 @@ RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+
 RewriteCond %{HTTP_ACCEPT} text/html [OR]
 RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
 RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
-RewriteRule ^$ https://swissartresearch.net/ontology/sari [R=303,L]
+RewriteRule ^$ https://schema.swissartresearch.net/ontology/sari [R=303,L]
 
 # Rewrite rule to serve RDF/XML content if requested
 RewriteCond %{HTTP_ACCEPT} application/rdf\+xml [OR]
 RewriteCond %{HTTP_ACCEPT} application/owl\+xml
-RewriteRule ^$ https://swissartresearch.net/ontology/sari.rdfs.xml [R=303,L]
+RewriteRule ^$ https://schema.swissartresearch.net/ontology/sari/onto.ttl [R=303,L]
```

Comment:
Did you mean to change this to .ttl?  There's an owl+xml conditional here.

Reply:
Oh yes, that was a mistake. I actually cleaned and changed a bit of things around, so I would actually would love to substitute the whole .htaccess with this one:

```suggestion
Options +FollowSymLinks
RewriteEngine on

# Turn off MultiViews
Options -MultiViews

AddType text/turtle .ttl

# In case of accept header <text/turtle>
RewriteCond %{HTTP_ACCEPT} ^.*text/turtle.* 
RewriteRule ^$ https://schema.swissartresearch.net/ontology/sari.ttl [R=303,NE,L]

# If suffix ttl, redirect to turtle version
RewriteRule ^sari.ttl$ https://schema.swissartresearch.net/ontology/sari.ttl [R=302,NE,L]

# If suffix html, redirect to html version
RewriteRule ^sari.html$ https://schema.swissartresearch.net/ontology/sari [R=302,NE,L]

# Default response: html
RewriteRule ^$ https://schema.swissartresearch.net/ontology/sari [R=303,NE,L]
```

should I make another pull?